src/org/labkey/targetedms/SkylinePort/Irt/RetentionTimeRegression.java

```diff
@@ -957,7 +957,8 @@ private void setIndex(int value)
 		{
 			privateIndex = value;
 		}
-		public int compareTo(DeltaIndex other)
+		@Override
+        public int compareTo(DeltaIndex other)
```

Comment:
Formatting looks mangled. Maybe it was pre-mangled and we should do a general reformat in this PR too?